config/locales/diaspora/en.yml

```diff
@@ -1008,8 +1008,13 @@ en:
 
   profiles:
     edit:
-      your_public_profile: "Your public profile"
-      your_private_profile: "Your private profile"
+      basic: "Basic"
+      additional: "Additional"
```

Comment:
One clarification: this PR introduces a flag to make either all of the details public or all non-public, it's not a per field setting.